CHANGELOG.md

```diff
@@ -10,9 +10,13 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ### Changed  
 
+- POST body for SCStormRunoffSyntheticUnitHydrograph for updated SCRunoffModelingServices
```

Comment:
I kind of think we should wait to add anything to the changelog until we are ready to merge with dev. I removed it when I merged with dev a couple weeks ago and there were a ton of merge conflicts in the changelog alone. 

Also, the changelog in this branch was getting really long with just little changes we were making related to anything SC. We were adding things to the fixed column that hadn't made it to dev. I think it makes more sense once we are ready to push to dev to just have the higher-level changes in the changelog, for example 'added Bohman rural method' & things like that.